int64 bee_id
